File: (1) Python/(1) Python Programming Basics - September 2022/(5) Conditional Statements Advanced - Lab/small_shop.py
product = input()
city = input()
number_of_products = float(input())

if city == "Sofia":
    if product == "coffee":
        print(0.5 * number_of_products)
    elif product == "water":
        print(0.8 * number_of_products)
    elif product == "beer":
        print(1.2 * number_of_products)
    elif product == "sweets":
        print(1.45 * number_of_products)
    elif product == "peanuts":
        print(1.6 * number_of_products)
elif city == "Plovdiv":
    if product == "coffee":
        print(0.4 * number_of_products)
    elif product == "water":
        print(0.7 * number_of_products)
    elif product == "beer":
        print(1.15 * number_of_products)
    elif product == "sweets":
        print(1.30 * number_of_products)
    elif product == "peanuts":
        print(1.5 * number_of_products)
elif city == "Varna":
    if product == "coffee":
        print(0.45 * number_of_products)
    elif product == "water":
        print(0.7 * number_of_products)
    elif product == "beer":
        print(1.1 * number_of_products)
    elif product == "sweets":
        print(1.35 * number_of_products)
    elif product == "peanuts":
        print(1.55 * number_of_products)

File: (3) Java/(01) Programming Basics with Java - January 2025/(05) Conditional Statements Advanced - Lab/small_shop.java
import java.util.Scanner;

public class small_shop {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        String product = scan.nextLine();
        String city = scan.nextLine();
        double quantity = Double.parseDouble(scan.nextLine());

        double priceOfProduct = 0;

        switch (city) {
            case "Sofia" -> {
                switch (product) {
                    case "coffee" -> priceOfProduct = 0.5;
                    case "water" -> priceOfProduct = 0.8;
                    case "beer" -> priceOfProduct = 1.2;
                    case "sweets" -> priceOfProduct = 1.45;
                    case "peanuts" -> priceOfProduct = 1.6;
                }
            }
            case "Plovdiv" -> {
                switch (product) {
                    case "coffee" -> priceOfProduct = 0.4;
                    case "water" -> priceOfProduct = 0.7;
                    case "beer" -> priceOfProduct = 1.15;
                    case "sweets" -> priceOfProduct = 1.3;
                    case "peanuts" -> priceOfProduct = 1.5;
                }
            }
            case "Varna" -> {
                switch (product) {
                    case "coffee" -> priceOfProduct = 0.45;
                    case "water" -> priceOfProduct = 0.70;
                    case "beer" -> priceOfProduct = 1.1;
                    case "sweets" -> priceOfProduct = 1.35;
                    case "peanuts" -> priceOfProduct = 1.55;
                }
            }
        }

        double totalPrice = priceOfProduct * quantity;
        System.out.println(totalPrice);
    }
}
